icu4c/source/common/unicode/docmain.h

```diff
@@ -138,6 +138,11 @@
  *     <td>icu::DateFormat</td>
  *   </tr>
  *   <tr>
+ *     <td>Relative Date and Time Formatting</td>
+ *     <td>reldatefmt.h</td>
```

Comment:
```suggestion
 *     <td>ureldatefmt.h</td>
```
This column is for the C API.